docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,79 @@
+@startuml
+'https://plantuml.com/sequence-diagram
```

Comment:
Can leave out this commented line? I'm guessing you just added it there for you to refer

Reply:
On second thoughts, leave it there. Everyone else might need it